0000-route-broadcasting-protocol/0000-route-broadcasting-protocol.md

```diff
@@ -162,6 +188,7 @@ Route Update is done in the following procedure:
 
 - A node sends a route update request to the corresponded node with information of route update logs.
 - The receiver node updates its routing table, and responds that the request is done.
+- If the receiving node is not configured to receive route updates, the receiving node responds error.
```

Comment:
Minor nit, should read: ...`the receiving node responds with an error`

Reply:
Fixed to use prepositions.